README.md

```diff
@@ -340,6 +341,12 @@ div("Argh!", bigFont)
 // <div style="font-size: 40px; font-weight: bold;">Argh!</div>
 ```
 
+If you want to reuse it, but want to overwrite a property:
```

Comment:
Maybe we can even be more specific. How about this?

"If you want to reuse the `bigFont`, but want to overwrite one of its properties, you can combine two `VDomModifier`. Here the latter `fontSize` will overwrite the one from `bigFont`:"